tqdm.py

```diff
@@ -0,0 +1,42 @@
+import cardinality  # You might need to do 'pip install cardinality'.
```

Comment:
Very cool that you searched for an external library, but...
This one is very suspicious - only 5 stars in GitHub, and not maintained at all (last commit 3 years ago).
I'd avoid using it, and just copy the code from there (if it suits me).
In our case, it doesn't, since it only adds support for these kind of cases:
`cardinality.count(i for i in range(500))`
Which were not requested in the exercise.